src/Steppenwolf/wwwroot/scss/style.scss

```diff
@@ -1,6 +1,6 @@
-﻿@import "../lib/bootstrap/scss/bootstrap";
-
-@import "open-iconic/font/css/open-iconic-bootstrap.min.css";
+﻿//@import "../lib/bootstrap/scss/bootstrap";
```

Comment:
Why are we removing the bootstrap from here?

Reply:
It is commented out to avoid having the styles 2 times. Reference - https://github.com/043Tech/Steppenwolf/issues/24